src/main/java/no/nav/fo/veilarbregistrering/httpclient/SykmeldtInfoClient.java

```diff
@@ -36,23 +30,16 @@ public InfotrygdData hentSykmeldtInfoData(String fnr) {
     }
 
     private InfotrygdData getSykeforloepMetadata(String url) {
-
-        String token;
         HttpServletRequest request = httpServletRequestProvider.get();
-
-        if (AutentiseringUtils.erInternBruker()) {
-            token = issoTokenLocator.getToken(request).orElse(null);
```

Comment:
trengs ikke issoTokenLocator lenger?

Reply:
høres logisk ut:)

Reply:
hvilke token er det som brukes veileder ? selvbetjening-idtoken for veileder?

Reply:
Siden veiledere ikke gjør kall mot infotrygd lenger, så trenger vi ikke noe FSS logikk i SykmeldtInfoCLient :)